### PR TITLE
Add manger spec to database unit test

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

**Comment by yanmxa** [comment]
**unit test with testcontainer-go**
```bash
$ export KUBEBUILDER_ASSETS="/root/.local/share/kubebuilder-envtest/k8s/1.24.2-linux-amd64"
$ go test ./manager/pkg/specsyncer/spec2db -v
=== RUN   TestSpec2db
Running Suite: Spec2db Suite - /root/workspace/hub-of-hubs/manager/pkg/specsyncer/spec2db
=========================================================================================
Random Seed: 1662346159

Will run 3 of 3 specs
2022/09/05 02:49:23 Starting container id: 36f85340e0b6 image: docker.io/testcontainers/ryuk:0.3.3
2022/09/05 02:49:24 Waiting for container id 36f85340e0b6 image: docker.io/testcontainers/ryuk:0.3.3
2022/09/05 02:49:24 Container is ready id: 36f85340e0b6 image: docker.io/testcontainers/ryuk:0.3.3
2022/09/05 02:49:24 Starting container id: 29943da79500 image: postgres:14.1-alpine
2022/09/05 02:49:24 Waiting for container id 29943da79500 image: postgres:14.1-alpine
2022/09/05 02:49:26 Container is ready id: 29943da79500 image: postgres:14.1-alpine
•••

Ran 3 of 3 Specs in 9.981 seconds
SUCCESS! -- 3 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestSpec2db (9.98s)
PASS
ok      github.com/stolostron/multicluster-global-hub/manager/pkg/specsyncer/spec2db    10.013s
```

File: test/pkg/postgre/main.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package main
```
Comment (clyang82) [inline test/pkg/postgre/main.go:1]: may use `postgres` instead of `postgre`.

**Comment by clyang82** [comment]
Do you know why there is `No Coverage information`?

File: Makefile [hunk 2]
```diff
@@ -56,8 +56,8 @@ setup_envtest:
 unit-tests-operator: setup_envtest
 	KUBEBUILDER_ASSETS="$(shell ${TMP_BIN}/setup-envtest use --use-env -p path)" ${GO_TEST} `go list ./operator/... | grep -v test`
 
-unit-tests-manager:
-	${GO_TEST} `go list ./manager/... | grep -v test`
+unit-tests-manager: setup_envtest
+	KUBEBUILDER_ASSETS="$(shell ${TMP_BIN}/setup-envtest use --use-env -p path)" ${GO_TEST} `go list ./manager/... | grep -v test`
```
Comment (morvencao) [inline Makefile:60]: life is too short to check the tailing whitespace, maybe we can add a linter for this in another PR.

**Comment by clyang82** [comment]
`No Coverage information` maybe due to you do not touch any non-test code.

/lgtm

**Pull request merged**